hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/SplitBrainTest.java

```diff
@@ -81,7 +76,7 @@ public void when_quorumIsLostOnMinority_then_jobRestartsUntilMerge() {
             MockPS processorSupplier = new MockPS(StuckProcessor::new, clusterSize);
             DAG dag = new DAG().vertex(new Vertex("test", processorSupplier));
             jobRef[0] = instances[0].newJob(dag, new JobConfig().setSplitBrainProtection(true));
-            assertOpenEventually(StuckProcessor.executionStarted);
+            assertOpenEventually(StuckProcessor.executionStarted, 10);
```

Comment:
why do we use a 10-sec hard limit for eventual assertions here? It can be quite problematic when the pr builder slows down. The default timeout must be just fine.